pkg/kubelet/lifecycle/handlers.go

```diff
@@ -187,20 +187,11 @@ func (a *noNewPrivsAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult
 		return PodAdmitResult{Admit: true}
 	}
 
-	// Always admit for remote runtime.
-	if a.Runtime.Type() == kubetypes.RemoteContainerRuntime {
+	// Always admit runtimes except docker.
+	if a.Runtime.Type() != kubetypes.DockerContainerRuntime {
```

Comment:
The following `if` is not needed, because it must be docker.

Reply:
good catch